app/config/services/services.xml

```diff
@@ -94,6 +94,7 @@
         <service id="AppBundle\Donation\PayboxFormFactory" />
 
         <!-- Membership -->
+        <service id="AppBundle\Command\RemindUnconfirmedMembershipCommand"/>
```

Comment:
You can declare this service in the `<!--Command-->` section, what do you think ?